src/main/scala/com/codacy/client/stash/service/UserServices.scala

```diff
@@ -72,4 +75,23 @@ class UserServices(client: StashClient) {
     client.delete(url)()
   }
 
+  /**
+    *
+    * We need this fallback for the cases when a username contains special characters because the
```

Comment:
This is not true right?
The problem is that sometimes you need to find the user by the email and not the username, right?

Reply:
Nop, the problem is that a user can have an email on the name or other case that makes a name having special characters

Reply:
Hmmm, where did you get that information from?

All the information I collected on the call was that `/plugins/servlet/applinks/whoami` returns sometimes the email other times the slug.
But if you request the user is always returns the correct things in the correct fields.